benchmarking/repo_driver.py

```diff
@@ -215,13 +215,14 @@ def _setupRepoStep(self, platform, commit):
 
     def _buildProgram(self, platform, repo_info):
         directory = getDirectory(repo_info['commit'], repo_info['commit_time'])
-
+        program_name = getArgs().framework + "_benchmark" + (".exe" if os.name == "nt" else "")
         dst = os.path.join(getArgs().exec_dir, getArgs().framework,
-            platform, directory,
-            getArgs().framework +
-            "_benchmark" + (".exe" if os.name == "nt" else ""))
-
+            platform, directory, program_name)
         repo_info["program"] = dst
+        filedir = os.path.dirname(dst)
+        for fn in os.listdir(filedir):
+            if fn != program_name:
+                repo_info[fn] = os.path.join(filedir, fn)
```

Comment:
You are adding everything in a directory to the dictionary. I feel it is a bit fragile. 
Also, instead of listing everything at the top level of the info struct, maybe create a field called "program", and list everything within that field?